### PR TITLE
Pc streamline secrets and convert for CS48

### PR DESCRIPTION
**Pull request by pconrad**
In this PR, we:
* Adopt Bryan's streamlined workflow for secrets, using properties files instead of JSON files
* Make selection of `localhost` vs. `heroku` profile automatic
* Change CS56 W20 package names and strings to CS48 S20
* Changed default admin list to only P. Conrad and S. Chow (removing other CS56 W20 TAs)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -225,4 +234,84 @@
...
+            <id>localhost</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/config/localhost</directory>
+                    </resource>
+                </resources>
```
Comment (btk5h) [inline pom.xml]: This config file doesn't exist anywhere, is it missing or are these lines unnecessary?

File: pom.xml [hunk 2]
```diff
@@ -225,4 +234,84 @@
...
+                <property>
+                    <name>env.HEROKU_PROPERTIES</name>
+                </property>
+            </activation>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/config/heroku</directory>
+                    </resource>
+                </resources>
```
Comment (btk5h) [inline pom.xml]: Same as above

**Review by btk5h — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (pconrad) [reply pom.xml]: Also will eliminate

File: pom.xml (hunk 1 above)
Comment (pconrad) [reply pom.xml]: Good call.  I'll eliminate
Have eliminated.

**Review by btk5h — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (pconrad) [reply pom.xml]: Also noted that h2 and postgres were in pom.xml twice: once in profile and once in main file.

Have kept only the version in the profile.

**Pull request merged**